gitlabform/gitlabform/processors/util/difference_logger.py

```diff
@@ -0,0 +1,53 @@
+import hashlib
+import json
+import logging
+from itertools import starmap
+
+
+class DifferenceLogger(object):
+    @staticmethod
+    def log_diff(
+        subject, from_config, from_server, only_changed=False, hide_entries=None
```

Comment:
Nit:

I know that [me-in-may-2019 suggested `from_config` and `from_server` parameter names](https://github.com/egnyte/gitlabform/pull/44/files#r280068447) but me-in-october-2020 had no idea what they are about until I read what I wrote there... 😅 

Do you think that `before` and `after`, in this order, would be better as these parameter names, @kowpatryk ?

Or maybe you have a completely different proposal?

Reply:
Hmm... how about `from_server` -> `current_config` and `from_config` -> `config_to_apply`? This would be quite descriptive what stands behind those parameters.

Reply:
Sure, that sounds good!